.github/pull-request-template.md

```diff
@@ -0,0 +1,62 @@
+### Describe your PR
+
+<!-- 
+Provide a clear and concise description of what your PR does. 
+This helps reviewers understand the purpose and context of your changes. 
+-->
+
+
+### What type of PR is this?
```

Comment:
As discussed in the call, let's remove this from the PR template and update the workflow to create comments on the PR when the workflow fails